CONTRIBUTING.md

```diff
@@ -4,15 +4,47 @@ This document provides information about contributing to the [Keptn documentatio
 This documentation is authored with [markdown](https://www.markdownguide.org/basic-syntax/)
 and rendered using the [Hugo static website generator framework](http://gohugo.io).
 
+We welcome and encourage contributions of all levels.
+You can make modifications using the github editor;
```

Comment:
The github editor doesn't work that well for keptn because we have the requirement for the DCO which is not available in the GH editor AFAIK

Reply:
I don't know if this can be fixed by replacing the DCO. It really depends on the replacement IMO.

Reply:
The DCO needs to be attached to every single commit you make, so there's no nice way to get around it. You can however add the DCO to the commit messages after the fact of course by doing a rebase of your branch.

Reply:
The DCO states: `By making a contribution to this project, I certify that: ....`

Keptn is therefore compliant by ensuring that "by making a contribution via PR to this project, you are implicitely agreeing to the DCO". Once and done?

Reply:
I fully agree that the DCO has no real purpose whatsoever since it's just some random text that is added somewhere. The problem is rather that for now, we have the requirement by the CNCF to have it...
So my suggestion would be to only document workflows that actually work for us right now.

Reply:
I think Oleg has something in mind that will fix this -- and other -- problems.  I added a mealy-mouth warning that using the github editor causes the CI to fail and I think that is good enough for now.  See if you agree.

Reply:
Yes, I am working on it. FTR https://github.com/keptn/community/issues/72